test/runtests.jl

```diff
@@ -111,5 +111,7 @@ cd("modelling")
         0.1         1.0]
     include("modelling/rocket_control.jl")
     @test objective_value(rocket) ≈ 1.0128340648308058
+
+    include("modelling/facility_location.jl")
```

Comment:
Do you know if that would be possible to test the solution returned by the solver? Or at least test if the return status of the solver is `MOI.OPTIMAL`?